lib/game.rb

```diff
@@ -0,0 +1,53 @@
+require './lib/board'
+
+class Game
+  attr_reader :message
+  attr_reader :current_board
+  def initialize; end
+
+  def start
+    @board = Board.new
+    @current_board = @board.display_board
+  end
+
+  # rubocop:disable Metrics/PerceivedComplexity
+  def validate_position(position, player)
```

Comment:
- [ ] **[OPTIONAL]** This method is doing too much and it can be seen from the rubocop alert. It should respect single responsibility principle, I suggest to break this method to a number of method to
- check if cell position is valid (valid_input?)
- check if cell position is already taken (cell_taken?)
Also it should not contain the `update_board` method, it should reflect the method name (validate_position) only for validating position (returning true or false)